contrib/ci/olsr-site/image-customization

```diff
@@ -0,0 +1,20 @@
+features {
+	'autoupdater',
+	'ebtables-filter-multicast',
+	'ebtables-filter-ra-dhcp',
+	'ebtables-limit-arp',
+	'mesh-olsrd',
+	'mesh-vpn-fastd',
+	'respondd',
+	'status-page',
+	'web-advanced',
+	'web-wizard',
+}
+
+packages({
```

Comment:
A lot of the examples are inconsistent regarding use of `{` vs. `({`. I'd prefer to use `{` to be consistent with package lists in target files.

Reply:
While we are at it, we had examples where we needed brackets:
https://github.com/ffac/site/commit/8cddeadc890b9e6b12ec3a347b875d5846919f79#diff-d79e4f28ca1dc9caa17a36e10310542744140aa58c9ecfd00256c741bca2d7afR133-R135

So I am not sure if a simple `{ 'a' }` is better than a `({ 'a' })` considering that some people use vars like `(list)`.

Reply:
I think I've fixed all inconsistencies with the latest push. Please check if you still find any.